person.rb

```diff
@@ -22,16 +25,22 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self
+  end
```

Comment:
- [ ] It would be better to use the Class `Rental` inside the add_rental  to create a rental and add the book and person automatically adding it the initialize method like the following code : 

```ruby
# in Book and Person Class
def add_rental(book, date)
    Rental.new(date, book, self)
end

# Rental Class
  def initialize(date, book, person)
    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
```
that way the belongs to the association will add automatically

Reply:
Thanks for the advice, I changed the methods mentiones